pages/resources/testnet.md

```diff
@@ -10,8 +10,8 @@ import EVMAssets from '../../components/evm/assets'
 
 | Variable              | Value     |
 | --------------------- | --------- |
-| `axelar-core` version | `v0.29.1` |
-| `vald` version        | `v0.29.1` |
+| `axelar-core` version | `v0.31.1` |
+| `vald` version        | `v0.31.1` |
```

Comment:
this is changed after/when the upgrade takes place (since our scripts read this page by default if a version isn't provided)